x-pack/plugins/index_management/server/routes/api/mapping/register_index_mapping_route.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RouteDependencies } from '../../../types';
+import { registerMappingRoute } from './register_mapping_route';
+import { registerUpdateMappingRoute } from './register_update_mapping_route';
+
+export function registerIndexMappingRoutes(dependencies: RouteDependencies) {
+  registerMappingRoute(dependencies);
```

Comment:
nit: this route could be renamed to "get mappings" route for clarity

Reply:
added in https://github.com/elastic/kibana/pull/178923/commits/fb98148ae91566ce3461bf4ba024624f44270cdb